### PR TITLE
Show Species & re-measurement data

### PR DESCRIPTION
**Pull request by prachigarg19**
- if no sample tree exists, don’t show sample trees title
- show remeasurement data 

Changes in this pull request:
- Display remeasurement values for all sample trees.

To test the functionality for remeasurement values, use HID:  EP0BTP 
Including species aliases have been reverted and will be tackled in a separate PR.

@

### CODE REVIEW

File: src/features/user/TreeMapper/components/PlantLocationPage.tsx [hunk 1]
```diff
@@ -92,6 +107,25 @@ export default function PlantLocationPage({
       </div>
 
       <LocationDetails {...DetailProps} />
+      <br />
+      <br />
+      {species && (
+        <>
+          <b>Sample Tree</b>
```
Comment (norbertschuler) [inline src/features/user/TreeMapper/components/PlantLocationPage.tsx:114]: Please extract test to resource for translations.

**Review by norbertschuler — CHANGES_REQUESTED** [review]

File: src/features/user/TreeMapper/components/PlantLocationPage.tsx [hunk 2]
```diff
@@ -92,6 +107,25 @@ export default function PlantLocationPage({
...
+          <div className={styles.sampleTreesContainer}>
+            {species.map((tree, index) => (
+              <div className={styles.singleTree} key={tree.id}>
+                <p>
+                  {index + 1} .{' '}
```
Comment (norbertschuler) [inline src/features/user/TreeMapper/components/PlantLocationPage.tsx:119]: Why not use an [ordered list](https://www.w3schools.com/html/html_lists.asp) with the `<ol>` tag?

**Review by norbertschuler — CHANGES_REQUESTED** [review]

**Comment by norbertschuler** [comment]
Can't test this currently as I have no RO with suitable test data.

File: src/features/user/TreeMapper/components/PlantLocationPage.tsx [hunk 3]
```diff
@@ -259,37 +259,50 @@ export function LocationDetails({
...
+                      </span>
+                      <br />
+                      {spl.status === 'dead' ? (
+                        <p>
+                          {spl?.status} • {spl?.statusReason}
+                        </p>
+                      ) : (
+                        ''
+                      )}
+                      {[...spl.history].reverse().map((h, index) => (
```
Comment (Shreyaschorge) [inline src/features/user/TreeMapper/components/PlantLocationPage.tsx:292]: Not sure if the history array will be sorted by created date.
Just to be on the safe side please use `moment` or `date-fns` library (which is already present in the project) to sort it by created date

**Review by Shreyaschorge — CHANGES_REQUESTED** [review]
Just a minor change.

File: src/features/user/TreeMapper/components/PlantLocationPage.tsx (hunk 3 above)
Comment (prachigarg19) [reply src/features/user/TreeMapper/components/PlantLocationPage.tsx:292]: Yes history array is supposed to be sorted according to design provided.

**Review by Shreyaschorge — APPROVED** [review]
LGTM!

**Comment by norbertschuler** [comment]
Fix the Git conflict, remove the `PR: draft` label and get this merged if ready ;-)

**Comment by prachigarg19** [comment]
> Fix the Git conflict, remove the `PR: draft` label and get this merged if ready ;-)

This PR needs to have species added by an RO as well along with their respective remeasurement data. This PR is on hold until the required data is created.

**Comment by mariahosfeld** [comment]
Closed in favour of different implementation.

**Pull request closed without merging**